docs/image.rst

```diff
@@ -39,8 +50,15 @@ Classes
 
     The other form creates an empty image with ``width`` columns and
     ``height`` rows. Optionally ``buffer`` can be an array of
-    ``width``×``height`` integers in range 0-9 to initialize the image.
-
+    ``width``×``height`` integers in range 0-9 to initialize the image:
+   
+        Image(2, 2, b'\x08\x08\x08\x08')
```

Comment:
Can we also add an example with a byte array?
```
Image(2, 2, bytearray([0,1,1,0]))
```

Reply:
done in https://github.com/bbcmicrobit/micropython/pull/582/commits/24b83b74c2cf1bddd8b9c9dcbcef9096979dd24f

Reply:
done in https://github.com/bbcmicrobit/micropython/pull/582/commits/24b83b74c2cf1bddd8b9c9dcbcef9096979dd24f

Reply:
done in https://github.com/bbcmicrobit/micropython/pull/582/commits/24b83b74c2cf1bddd8b9c9dcbcef9096979dd24f